11-Reset/readme.md

```diff
@@ -0,0 +1,67 @@
+git-reset
+=========
+
+> Reset current HEAD to the specified state
+
+Git reset has three forms. &lt;tree-ish&gt; defaults to HEAD in all of them.
+
+git reset [-q] [<tree-ish>] [--] <paths>...
+-------------------------------------------
+
+> Can be thought of as opposite for `git-add`.
+
+Resets index entries for all &lt;paths&gt; to their state at &lt;tree-ish&gt;. Important part here is that this form of `git-reset` doesn't affect the working tree or current branch in any form.
```

Comment:
Please see the exact for of this reset, it doesn't have soft/hard modes. Resets with modes are in section bellow.